pyroll/freiberg_flow_stress/__init__.py

```diff
@@ -27,4 +41,24 @@ def f(strain: float, strain_rate: float, temperature: float) -> float:
         return f
 
 
-from . import materials
+try:
+    @DeformationUnit.Profile.pillars_flow_stress
+    def pillars_flow_stress(self: DeformationUnit.Profile):
+        if hasattr(self, "freiberg_flow_stress_coefficients"):
+            return flow_stress(
+                self.freiberg_flow_stress_coefficients,
+                self.pillar_strains,
+                self.unit.pillar_strain_rates,
+                self.temperature
+            )
+
+    def initial_values(self: RollPass):
+        from pyroll.pillar_model import Config
+        self.in_profile.pillar_strains = np.full(Config.PILLAR_COUNT, 0.1)
```

Comment:
Where does this `0.1` come from?

Reply:
Was seen as sort of base_strain according to the model but your right this is set in the function itself. 
So as I see it now we can set is to 0. What do you think?

Reply:
yes, like `strain` on in profile to 
```suggestion
        self.in_profile.pillar_strains = np.zeros(Config.PILLAR_COUNT)
```